zerver/actions/invites.py

```diff
@@ -31,15 +31,15 @@
     Realm,
     Stream,
     UserProfile,
+    active_user_ids,
     filter_to_valid_prereg_users,
 )
 from zerver.tornado.django_api import send_event
 
 
 def notify_invites_changed(realm: Realm) -> None:
     event = dict(type="invites_changed")
-    admin_ids = [user.id for user in realm.get_admin_users_and_bots()]
-    send_event(realm, event, admin_ids)
+    send_event(realm, event, active_user_ids(realm.id))
```

Comment:
Should we just sent the event to admins and the user who would be affected like the one who sent the invite only and not all the non-admin users?

Reply:
Yes we can do that. So I think we should change the function to something like this
```
def notify_invites_changed(realm: Realm, acting_user: Optional[UserProfile]=None) -> None:
    event = dict(type="invites_changed")
    admin_ids = [user.id for user in realm.get_admin_users_and_bots()]
    recipient_ids=admin_ids
    if(acting_user):
        recipient_ids.append(acting_user.id)
   send_event(realm, event, recipient_ids)
```

Reply:
Yes, that seems correct.